mapbox/src/main/java/org/odk/collect/mapbox/PolyFeature.kt

```diff
@@ -93,7 +93,7 @@ internal class PolyFeature(
     private fun updateLine() {
         val points = mapPoints
             .map {
-                Point.fromLngLat(it.lon, it.lat, it.alt)
```

Comment:
It is confusing that Point.fromLngLat takes `longitude` as the first parameter whereas in usually it's the other way around and `latitude` should be first. Could you fix this?

Reply:
Not our method! Agreed, I'd typically have those the other way around, but this is a method on Mapbox's `Point` class.

Reply:
Ah right.